recipes/pysqa/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pysqa" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d8d07a0e8a63bf04237806d3cd1a4dc4e3613230109c486b289e23c87cfb155b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
You probably only need `python` and `pip` here. These are only the dependencies that are needed to run the `setup.py` file.